docs/source/fsdp.rst

```diff
@@ -0,0 +1,7 @@
+.. currentmodule:: torch.distributed.fsdp
+
+FullyShardedDataParallel
+========================
+
+.. autoclass:: torch.distributed.fsdp.FullyShardedDataParallel
+  :members:
```

Comment:
We will need an empty new line here I think, otherwise, we will get

```
docs/source/fsdp.rst
The above files do not have correct trailing newlines; please normalize them
Error: Process completed with exit code 1.

```